webapp/src/containers/MasternodesPage/components/MasterNodeTabHeader/index.tsx

```diff
@@ -8,10 +8,11 @@ import { ALL, MINE } from 'src/constants';
 interface MasterNodeTabsHeaderProps {
   setTab: (tab: string) => void;
   tab: string;
+  setFooterHide: (boolean) => void;
 }
 
 const MasterNodeTabsHeader = (props: MasterNodeTabsHeaderProps) => {
-  const { setTab, tab } = props;
+  const { setTab, tab, setFooterHide } = props;
```

Comment:
hi @Stonygan, wanted to ask why did you add this setFooterHide for some methods? thanks